en/docs/administer/blue-green-data-encryption-keyrotation.md

```diff
@@ -0,0 +1,252 @@
+    | IDN_OAUTH2_ACCESS_TOKEN       | OAuth 2.0 tokens                                        | Need to sync if the tokens created during the key rotation period needs to be valid after key rotation.                                                                         |
+    | IDN_OAUTH2_ACCESS_TOKEN_SCOPE | OAuth 2.0 scopes                                        | If the IDN_OAUTH2_ACCESS_TOKEN is synced, this table also needs to be synced.                                                                                            |
+    | IDN_OAUTH2_AUTHORIZATION_CODE | OAuth 2.0 authorization codes                           | Need to sync if the authorization codes created during the key rotation period need to be valid after key rotation. |
+
+
+!!! note
+    In this section, `<OLD_IS_HOME>` is the directory where the current Identity Server resides in, and `<NEW_IS_HOME>` is the directory where the copy of the current Identity Server resides in. `<KEY_ROTATION_REPO>` refers to the location [here](https://github.com/wso2/identity-tools/components/org.wso2.carbon.identity.keyrotation) and the `<KEY_ROTATION_TOOL>` refers to the location of the external tool
+
+## How To Create The Key Rotation Tool
+
+1.  Clone the repository [identity-tools](https://github.com/wso2/identity-tools)
+
+2.  Built it using maven by running the command `mvn clean install`
+
+3.  Go to the `<KEY_ROTATION_REPO>/target` folder and copy the `keyrotation-tool-<version>-SNAPSHOT.jar` jar file and the `<KEY_ROTATION_REPO>/target/lib` folder to `<KEY_ROTATION_TOOL>` location. Get the `properties.yaml` file, `keyrotation.sh` file and the `triggers` folder from `<KEY_ROTATION_REPO>/src/main/resources` and copy to the same `<KEY_ROTATION_TOOL>` location.
+  
+## Performing blue-green key rotation
+
+1.  Block all privileged user flows and allow only end user flows.
+
+    !!! note
+        For the privileged user flows, block all admin services from the load balancer and the management console as well. For end user flows the above [tables](../../administer/blue-green-data-encryption-keyrotation/#tables-that-support-syncing) will be synced to the `<NEW_IS_HOME>`, so only these end user data flows should be allowed to generate in the `<OLD_IS_HOME>`.
+
+2.  Execute the `old<identity-db-type>.sql` script in the `<OLD_IS_HOME>` identity database to create temp tables and triggers
+
+    !!! note
+        The triggers can be found inside `<KEY_ROTATION_TOOL>/triggers` folder.
+
+3.  Create a copy of the `<OLD_IS_HOME>`(`<NEW_IS_HOME>`) and dump `<OLD_IS_HOME>` identity and registry databases and create the new databases
+
+4.  Drop the temp tables and triggers in the `<NEW_IS_HOME>` identity database using the `new<identity-db-type>.sql` script
+
+    !!! note
+        The triggers can be found inside `<KEY_ROTATION_TOOL>/triggers` folder.
```

Comment:
```suggestion
        The triggers can be found inside the `<KEY_ROTATION_TOOL>/triggers` folder.
```

Reply:
Addressed with https://github.com/wso2/docs-is/pull/2555/commits/e4826286f4b66bb74b49890f19dce24148b95808